dotcom-rendering/src/lib/ad-targeting.ts

```diff
@@ -1,3 +1,14 @@
+import { isString } from '@guardian/libs';
+
+const getUrlKeywords = (url: string): string[] => {
+	const lastSegment = url
+		.split('/')
+		.filter(Boolean) // This handles a trailing slash
+		.slice(-1)[0];
+
+	return isString(lastSegment) ? lastSegment.split('-').filter(Boolean) : [];
+};
```

Comment:
This method was copied from [commercial-core](https://github.com/guardian/commercial-core/blob/main/src/targeting/content.ts#L91-L98) because it isn't exported there. This felt okay but I'm open to feedback

Reply:
Maybe we could add a card to export it, and a TODO here to import it and remove the duplicate?

Reply:
Good idea, I'll add!